src/concepts/components/properties.md

```diff
@@ -1,26 +1,551 @@
 ---
-description: Parent to child communication
+description: The component could expose property attributes to receive data from the parent html component
 ---
 
 # Properties
 
-Properties enable child and parent components to communicate with each other.
+It is a common need to receive data from the parent component using properties.
 
-## Derive macro
+The properties are defined in a dedicated Rust struct derivating `Properties` and `Clone`traits.
```

Comment:
"The properties are defined in a dedicated Rust struct" => "a component's properties should be defined using a separate struct which derives both the `Properties` and `Clone` traits (the trait `Properties` requiring that the trait `Clone` is implemented for all types for which it is implemented."

Reply:
I update the doc in this way:
*-*-*
A component's properties should be defined using a separate struct which derives `Properties` and `Clone` traits.

> Trait `Properties` requires that the trait `Clone` is implemented for all types for which it is implemented.
*-*-*

Let me know if you still prefer your initial feedback.

Reply:
"Trait `Properties`" => "The `Properties` trait"
"trait `Clone`" => `Clone` trait
"for which it is implemented" => "for which `Properties` is derived"